src/components/header/Backtrace.css

```diff
@@ -0,0 +1,25 @@
+.backtrace {
+  margin: 5px;
+  padding: 0;
+  list-style: none;
+  display: block;
+  max-height: 25em;
+  overflow: auto;
+}
+
+.backtraceStackFrame {
+  margin: 0;
+  padding: 0;
+  list-style: none;
+  display: block;
+  max-width: 30em;
+  text-overflow: ellipsis;
```

Comment:
not for this patch, but we need a way to have the ellipsis at the middle. For example, extract the hostname and display it, and then display the end of the path only. We don't really need the scheme here, I think.

Reply:
Yeah, that's good to keep in mind.